pycaret/containers/models/time_series.py

```diff
@@ -15,7 +15,7 @@
 import pandas as pd
 import logging
 
-from sktime.forecasting.base._sktime import _SktimeForecaster  # type: ignore
+from sktime.forecasting.base import BaseForecaster  # type: ignore
```

Comment:
just for curiosity, does BaseForecaster has different default methods than _SktimeForecaster ?

Reply:
I need to double check this, but this is what they have done in their library as well (sktime)

Reply:
_OptionalForecastingHorizonMixin was also removed if I remember correctly.